### PR TITLE
Develop rm getrequest

### PR DESCRIPTION
**Pull request by rmarchandTen24**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/6559)

### CODE REVIEW

File: org/Hibachi/HibachiControllerREST.cfc [hunk 1]
```diff
@@ -53,6 +53,13 @@ component output="false" accessors="true" extends="HibachiController" {
 
         getFW().setView("public:main.blank");
         arguments.rc.headers["Content-Type"] = "application/json";
+        var functionName = getFW().getItem();
+        if(
+            len(functionName) >= 3 
+            && left(functionName,3)=='get'
+        ){
+            arguments.rc.context = 'GET';
```
Comment (jasperboyd) [inline org/Hibachi/HibachiControllerREST.cfc]: @rmarchandTen24 do we want to set to POST in the else?

Comment (rmarchandTen24) [reply org/Hibachi/HibachiControllerREST.cfc]: @jasperboyd context on a post is usually a process method so we can't

Comment (jasperboyd) [reply org/Hibachi/HibachiControllerREST.cfc]: that wouldn't be `rc.processContext`?

Comment (rmarchandTen24) [reply org/Hibachi/HibachiControllerREST.cfc]: https://github.com/ten24/slatwall/blob/develop/org/Hibachi/HibachiControllerREST.cfc#L753

**Pull request merged**